autodoc-admin-api/data.lua

```diff
@@ -189,7 +191,78 @@ return {
           ]],
         },
       }
-    }
+    },
+    config = {
+      title = [[Db-less configuration]],
+      description = "",
+      ["/"] = {
+        POST = {
+          title = [[Update db-less config]],
+          endpoint = [[<div class="endpoint post">/config</div>]],
+          description = [[
+            This endpoint allows resetting a db-less Kong with a new
+            yml data file.
```

Comment:
this uses yml, next paragraph uses yaml. Better use "declarative configuration" instead.